components/datadog/agent/docker.go

```diff
@@ -58,20 +60,30 @@ func NewDockerAgent(e config.CommonEnvironment, vm *remoteComp.Host, manager *do
 }
 
 func dockerAgentComposeManifest(agentImagePath string, apiKey pulumi.StringInput, envVars pulumi.Map) docker.ComposeInlineManifest {
+	runInPrivileged := false
+	for k, _ := range envVars {
+		if strings.HasPrefix(k, "DD_SYSTEM_PROBE_") {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review. Do look for something to sincerely praise. -->
**👏 praise**
🎉